specification/batch/data-plane/Microsoft.Batch/stable/2022-01-01.15.0/BatchService.json

```diff
@@ -12670,12 +12686,17 @@
           "title": "The priority of Jobs created under this schedule.",
           "description": "Priority values can range from -1000 to 1000, with -1000 being the lowest priority and 1000 being the highest priority. The default value is 0. This priority is used as the default for all Jobs under the Job Schedule. You can update a Job's priority after it has been created using by using the update Job API."
         },
+        "allowTaskPreemption": {
+          "type": "boolean",
+          "title": "Whether Tasks in this job can be preempted by other high priority jobs",
+          "description": "If the value is set to True other high priority jobs submitted to the system will take precedence and will be able requeue tasks from this job. You can update a job's allowTaskPreemption after it has been created using the update job API."
```

Comment:
Missing comma: "If the value is set to True,"

Reply:
This is a bit out of scope of this PR as it affects other properties too, but it seems weird to call out every updatable property in the description individually.